content/en/docs/faq.md

```diff
@@ -385,3 +385,9 @@ to help you migrate to Flux v2:
 - [How to manage multi-tenant clusters with Flux v2](https://github.com/fluxcd/flux2-multi-tenancy)
 - [Migrate from Helm Operator to Flux v2](/docs/migration/helm-operator-migration/)
 - [How to structure your HelmReleases](https://github.com/fluxcd/flux2-kustomize-helm-example)
+
+## Release questions
+
+### Where can I find information on how long versions are supported and how often to expect releases?
+
+Releases are as needed for now and a release page with the schedule will be created when Flux reaches general availability status. We understand the importance of having a release schedule and are aware that such a schedule will support the users' ability to plan for an upgrade in advance.
```

Comment:
```suggestion
Releases are as needed for now and a release page with the schedule will be created when Flux reaches general availability status. We understand the importance of having a release schedule and are aware that such a schedule will support the users' ability to plan for an upgrade in advance.

```

Reply:
Maybe point to `/roadmap` for now as well to at least point to some available information?

Maybe also point out that `flux2` in 2021 had at least monthly releases, often more and all controllers together 100+ releases in that time-frame? That at least would demonstrate some of the release velocity and while not a strict guarantee - at least an indication of progress (even if in a "pre-release" kind of fashion still).

Reply:
We can say that the release cadence is at least once per month.

Reply:
Let me know if the update looks good! If y'all think I should add in more info about our current releases let me know! Thanks!!